pinot-plugins/pinot-file-system/pinot-s3/pom.xml

```diff
@@ -65,55 +65,4 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
-
-  <profiles>
-    <profile>
-      <id>build-shaded-jar</id>
-      <activation>
-        <property>
-          <name>skipShade</name>
```

Comment:
Do we want to keep this option in the main pom?

Reply:
I don't see the usage for this for a while.